client/src/components/Typewriter.tsx

```diff
@@ -18,7 +18,7 @@ const Typewriter: React.FC<TypewriterProps> = ({ conversation, onMessageUpdate }
           setCurrentMessage(newMessage);
           onMessageUpdate(newMessage);  
           setCharIndex((prev) => prev + 1);
-        }, 50); // Adjust speed here
+        }, 25); // Adjust speed here
```

Comment:
Still need to load this from the message! Maybe add as issue? And with that CR you should adjust the default in server/response.py to a good value.